lib/World.d.ts

```diff
@@ -236,21 +247,21 @@ type QueryResult<T extends ComponentBundle> = Query<T> & {
 	This is used for many repeated random access to an entity. If you only need to iterate, just use a query.
 
 	```ts
-	local inflicting = world:query(Damage, Hitting, Player):view()
-	for (const [_, source] of world.query(DamagedBy)) {}
-		local damage = inflicting:get(source.from)
-	end
+	const inflicting = world.query(Damage, Hitting, Player).view()
+	for (const [_, source] of world.query(DamagedBy)) {
+		const damage = inflicting.get(source.from)
+	}
 
-	for _ in world:query(Damage):view() do end -- You can still iterate views if you want!
+	for (const [, damage] of world.query(Damage).view()) {} -- You can still iterate views if you want!
 	```
 	
 	@returns View See [View](/api/View) docs.
 	 */
-	view: View<T>;
+	view: () => View<T>;
```

Comment:
View is already an iterable function so it should not return it.

Reply:
It does not work without it